src/ansi-c/cprover_library.cpp

```diff
@@ -101,16 +104,16 @@ void cprover_c_library_factory(
     return;
 
   std::string library_text;
-
-  library_text=get_cprover_library_text(functions, symbol_table);
+  library_text = get_cprover_library_text(functions, symbol_table, false);
```

Comment:
While at it: I don't see why declaration and definition should be separate instructions here, can you please merge them?

Reply:
fixed